src/encoder.rs

```diff
@@ -2948,17 +2948,29 @@ fn encode_tile_group<T: Pixel>(
     let pre_cdef_frame = fs.rec.clone();
 
     /* TODO: Don't apply if lossless */
-    let rec = Arc::make_mut(&mut fs.rec);
     if fi.sequence.enable_cdef {
-      cdef_filter_frame(fi, rec, &blocks);
+      ti.tile_iter_mut(fs, &mut blocks)
```

Comment:
For purposes of CDEF application, instead of using the existing tileinfo, it might make sense to create a new one. There is no reason that we need to use the same number of tiles for CDEF as we do for the rest of the encode - in fact you might often want to use more tiles for CDEF because it has no compression loss (unlike bitstream tiles).

Reply:
Yes. I will create new tileinfo and decide number of tiles independent of the number for encode.